assets/app-preact/config.json

```diff
@@ -1,32 +1,26 @@
 {
   "packageManifest": {
-    "@pika/web": {
-      "webDependencies": [
-        "preact",
-        "preact-compat",
-        "preact-router"
-      ]
+    "snowpack": {
+      "webDependencies": ["preact"]
```

Comment:
I just realized trying `--include 'src/**/*.ts'` that Snowpack doesn’t support TypeScript! A good feature request that should come soon 🙂